### PR TITLE
Improve our README

### PR DESCRIPTION
**Pull request by jrjohnson**
Reorganized slightly and better focused on using yarn to run commands
which doesn't require installing ember-cli globally.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview for *ilios-frontend* ready!

Built with commit 0294b77fa7eeb85e5bd55a327c6de38325990be0

https://deploy-preview-3799--ilios-frontend.netlify.com

File: README.md [hunk 1]
```diff
@@ -7,58 +8,65 @@
 [![Slack](https://ilios-slack.herokuapp.com/badge.svg)](https://ilios-slack.herokuapp.com/)
 [![Stories in Ready](https://badge.waffle.io/ilios/frontend.png?label=ready&title=Ready)](https://waffle.io/ilios/frontend)
 
-## Web interface for accessing and managing [Ilios Platform](https://github.com/ilios/ilios) data.
+## Open Source Heroes
+
+Every day these for profit companies make developing Ilios possible.  Without the free tier of service they offer to
```
Comment (thecoolestguy) [inline README.md]: "for-profit"

File: README.md [hunk 2]
```diff
@@ -7,58 +8,65 @@
...
+open source software we would be nowhere so if you have a chance to pay them for something please do.
```
Comment (thecoolestguy) [inline README.md]: "open source software, Ilios would be nowhere, so if you have a chance to pay them for something, please do!"

File: README.md [hunk 3]
```diff
@@ -7,58 +8,65 @@
...
+
+### Installation
 
-We recommend installing Node.js though [NVM](https://github.com/creationix/nvm#installation)
+* `git clone git@github.com:ilios/frontend.git` this repository
+* `cd ilios-frontend`
+* `yarn install`
 
-#### Installation
+### Running / Development
```
Comment (thecoolestguy) [inline README.md]: Once it is up and running, the Frontend will need to know where your backend API server is located.  You can set this information in a system wide environment variable name `ILIOS_FRONTEND_API_HOST` or can just add the setting to a file named `.env` file within the same directory as your Ilios Frontend code:

`echo "ILIOS_FRONTEND_API_HOST=https://ilios3-demo.ucsf.edu" > .env`

**Comment by jrjohnson** [comment]
Thanks @thecoolestguy - I made those changes, but modified the Running / Development section a bit, can you take another look?

**Pull request merged**